### PR TITLE
doc(browsers): Add page with instructions for Brave Shields

### PR DESCRIPTION
**Pull request by fvsch**
Goal is to have a page that we can link to on GitHub with visual explanations of what to do to allow third-party service workers.

Related to #14 but does not close it: we need to also find ways to direct Brave users to this page.

### CODE REVIEW

**Comment by fvsch** [comment]
Can be reviewed by reading the rendered page at:
https://github.com/stackblitz/webcontainer-core/blob/2e5bc8c298dcc982a251411fa21969a04bff222b/browsers/brave/brave.md

@KwintenP @SamVerschueren Want to check the technical bits?

File: browsers/brave/brave.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## How to allow third-party service workers for stackblitz.com
+
+1. Visit a WebContainer-based project, for instance https://www.stackblitz.com/edit/nextjs. The project’s boot sequence might stay stuck on the “Running start command” step:
+
+<img src="1-stuck-project.png" width="784" alt="Screenshot of Brave on a WebContainer project with the Brave Shields feature on. Loading the project’s web server is stuck on the last step.">
+
+2. Click on the “Shields” icon at the right of the address bar, then click on “Advanced View”.
+
+<img src="2-shields-popup.png" width="380" alt="">
```
Comment (SamVerschueren) [inline browsers/brave/brave.md:19]: Maybe fill in the alt? Same for screenshot 3.

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
One minor improvement 👍 .

File: browsers/brave/brave.md (hunk 1 above)
Comment (fvsch) [reply browsers/brave/brave.md:19]: I thought about it, but the alt text would be redundant with the text before, so it's better to leave it empty in that case.

Empty alt text indicates that an image is decorative or redundant with the textual information in its context.

Comment (fvsch) [reply browsers/brave/brave.md]: You know what, we could remove all alt text in this page or have alt text for all images, both work IMO. It's a style question, minimal vs more verbose.

Let’s try alt text for all images.

**Comment by fvsch** [comment]
@SamVerschueren I also switched the Opera logo for Brave on the main README, and linked to this new page there.
I think showing Brave rather than Opera makes sense given the market share of both browsers with our target audience.

**Comment by jonathansampson** [comment]
Nice work on the README for Brave!

**Pull request merged**